### PR TITLE
Added NextJS and AngularJS

### PR DESCRIPTION
**Pull request by Delvitron1019**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,8 +170,17 @@ API
 
 NextJs
 
-1.  [NextJS](https://www.youtube.com/watch?v=1WmNXEVia8I).
-2.  [documentry ](https://www.w3schools.io/learn/nextjs-tutorial/).
+1.  [documentation](https://www.w3schools.io/learn/nextjs-tutorial/).
```
Comment (aaryan2134) [inline README.md:173]: Please make the first letter capital

Comment (aaryan2134) [reply README.md:173]: Also, please write in same format as "Angular.js Documentation"

**Review by aaryan2134 — CHANGES_REQUESTED** [review]

**Comment by Delvitron1019** [comment]
I have made the changes as per your request.

image

**Comment by aaryan2134** [comment]
Awesome!
Thank you for your contribution

**Pull request merged**